README.md

```diff
@@ -1,98 +1,242 @@
+
+Once the PR is closed or merged, the respective app is deleted again.
+
+```yaml
+name: App Platform Preview
+
+on:
+  pull_request:
+    branches: [main]
+
+permissions:
+  pull-requests: write
+
+jobs:
+  test:
+    name: preview
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Deploy the app
+        id: deploy
+        uses: digitalocean/app_action/deploy@main
+        with:
+          deploy_pr_preview: "true"
+          token: ${{ secrets.DIGITALOCEAN_ACCESS_TOKEN }}
+      - uses: actions/github-script@v7
+        env:
+          BUILD_LOGS: ${{ steps.deploy.outputs.build_logs }}
+          DEPLOY_LOGS: ${{ steps.deploy.outputs.deploy_logs }}
+        with:
+          script: |
+            const { BUILD_LOGS, DEPLOY_LOGS } = process.env
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: `:rocket: :rocket: :rocket: The app was successfully deployed at ${{ fromJson(steps.deploy.outputs.app).live_url }}.`
+            })
+      - uses: actions/github-script@v7
+        if: failure()
+        with:
+          script: |
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: `The app failed to be deployed. Logs can be found [here](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}).
+
+              ## Logs
+              <details>
+              <summary>Build logs</summary>
+
+              \`\`\`
+              ${BUILD_LOGS}
+              \`\`\`
+              </details>
+
+              <details>
+              <summary>Deploy logs</summary>
+
+              \`\`\`
+              ${DEPLOY_LOGS}
+              \`\`\`
+              </details>`
+            })
+```
+
+```yaml
+name: Delete Preview
+
+on:
+  pull_request:
+    types: [ closed ]
+
+jobs:
+  closed:
+    runs-on: ubuntu-latest
+    steps:
+      - name: delete preview app
+        uses: digitalocean/app_action/delete@main
+        with:
+          from_pr_preview: "true"
+          ignore_not_found: "true"
+          token: ${{ secrets.DIGITALOCEAN_ACCESS_TOKEN }}
+```
+
+## Note for handling container images
+
+It is strongly suggested to use image digests to identify a specific image like in the example above. If that is not possible, it is strongly suggested to use a unique and descriptive tag for the respective image (not `latest`).
+
+## Upgrade from v1.x
+
+The v1 branch of this action is no longer under active development. Its documentation is [still available](README_v1.md) though.
```

Comment:
Instead of having a README_v1.md, should we create a v1 branch and link to there?
I think the advantage of having a v1 branch is if someone is debugging a problem with v1 action, it's easy to get to the source.

Reply:
Duh. Of course... my brain is still in vacation it seems 😅 . Good callout, I'll do that.

Reply:
Done.